### PR TITLE
ddev: add page

### PR DESCRIPTION
**Pull request by reiniiriarios**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.20

### CODE REVIEW

File: pages/common/ddev.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`ddev logs -f`
+
+- Run composer within the container:
+
+`ddev composer`
+
+- Install Node.js version 12:
+
+`ddev nvm install 12`
```
Comment (CleanMachine1) [inline pages/common/ddev.md:24]:
```suggestion
`ddev nvm install {{version}}`
```

File: pages/common/ddev.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Follow the log trail:
+
+`ddev logs -f`
+
+- Run composer within the container:
+
+`ddev composer`
+
+- Install Node.js version 12:
```
Comment (CleanMachine1) [inline pages/common/ddev.md:22]:
```suggestion
- Install a specific Node.js version
```

File: pages/common/ddev.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`ddev nvm install 12`
+
+- Export a database:
+
+`ddev export-db --file=/tmp/db.sql.gz`
+
+- Update the container's apt package lists:
+
+`ddev exec sudo apt-get update`
```
Comment (CleanMachine1) [inline pages/common/ddev.md:32]:
```suggestion
`ddev exec {{echo 1}}`
```

File: pages/common/ddev.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Install Node.js version 12:
+
+`ddev nvm install 12`
+
+- Export a database:
+
+`ddev export-db --file=/tmp/db.sql.gz`
+
+- Update the container's apt package lists:
```
Comment (CleanMachine1) [inline pages/common/ddev.md:30]:
```suggestion
- Run a specific command within a container:
```

File: pages/common/ddev.md (hunk 3 above)
Comment (reiniiriarios) [reply pages/common/ddev.md]: Awesome notes, thanks! 👍🏻

File: pages/common/ddev.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`ddev composer`
+
+- Install a specific Node.js version:
+
+`ddev nvm install {{version}}`
+
+- Export a database:
+
+`ddev export-db --file=/tmp/db.sql.gz`
```
Comment (pixelcmtd) [inline pages/common/ddev.md:28]:
```suggestion
`ddev export-db --file={{/tmp/db.sql.gz}}`
```

File: pages/common/ddev.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Start up a project:
+
+`ddev start`
+
+- Configure a project's type and docroot:
+
+`ddev config`
+
+- Follow the log trail:
```
Comment (kbdharun) [inline pages/common/ddev.md:14]:
```suggestion
- [f]ollow the log trail:
```
Just a small suggestion

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**